<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>420</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>PM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>570</width>
     <height>420</height>
    </size>
   </property>
   <widget class="QLabel" name="darkBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>260</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 32, 35);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pmLogo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pmVersion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(219, 219, 219);</string>
    </property>
    <property name="text">
     <string>PM v1.0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pmAbout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(219, 219, 219);</string>
    </property>
    <property name="text">
     <string>A password manager is a computer program that allows users to store, generate, and manage their passwords for local applications and online services. A password manager assists in generating and retrieving complex passwords, storing such passwords in an encrypted database, or calculating them on demand. </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lightBg">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>0</y>
      <width>311</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 52);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pmTitle">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(219, 219, 219);</string>
    </property>
    <property name="text">
     <string>PM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="webLine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 52);
border:2px solid rgba(49, 49, 52);
border-bottom-color:rgba(129,230,217,255);
color: rgb(255, 255, 255);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Website</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 52);
border:2px solid rgba(49, 49, 52);
border-bottom-color:rgba(129,230,217,255);
color: rgb(255, 255, 255);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Email/User Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#searchButton{
	background-color: rgb(117,  214,  209);
	color:rgb(36,36,36);
	border-radius:5px;
    }
QPushButton#searchButton:pressed{
     padding-left:5px;
     padding-top:5px;
     background-position:calc(100% - 10px)center;
     }
QPushButton#searchButton:hover{
     background-color:rgba(174,  235,  232);
     }</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 52);
border:2px solid rgba(49, 49, 52);
border-bottom-color:rgba(129,230,217,255);
color: rgb(255, 255, 255);
padding-bottom:7px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#generateButton{
	background-color: rgb(117,  214,  209);
	color:rgb(36,36,36);
	border-radius:5px;
    }
QPushButton#generateButton:pressed{
     padding-left:5px;
     padding-top:5px;
     background-position:calc(100% - 10px)center;
     }
QPushButton#generateButton:hover{
     background-color:rgba(174,  235,  232);
     }</string>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#saveButton{
	background-color: rgb(117,  214,  209);
	color:rgb(36,36,36);
	border-radius:5px;
    }
QPushButton#saveButton:pressed{
     padding-left:5px;
     padding-top:5px;
     background-position:calc(100% - 10px)center;
     }
QPushButton#saveButton:hover{
     background-color:rgba(174,  235,  232);
     }</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="pmAlert">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="passwordEchoType">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#passwordEchoType:pressed{
	 background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/hidden.png</normaloff>img/hidden.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
